LSP-pyright.sublime-settings

```diff
@@ -12,47 +14,54 @@
 	// @see https://github.com/microsoft/pyright/blob/master/docs/settings.md
 	// @see https://github.com/microsoft/pyright/blob/master/client/package.json
 	"settings": {
-		// Offer auto-import completions.
-		"python.analysis.autoImportCompletions": true,
-		// Automatically add common search paths like 'src'?
-		"python.analysis.autoSearchPaths": true,
-		// Additional import search resolution paths
-		"python.analysis.extraPaths": [
-			// For Sublime Text plugin development
-			"$sublime_py_files_dir",
-			"$packages",
-		],
-		// Path to directory containing custom type stub files.
-		"python.analysis.stubPath": "./typings",
-		// "openFilesOnly" or "workspace"
-		"python.analysis.diagnosticMode": "openFilesOnly",
-		// Allows a user to override the severity levels for individual diagnostics.
-		// @see https://github.com/microsoft/pyright/blob/master/docs/configuration.md#type-check-diagnostics-settings
-		"python.analysis.diagnosticSeverityOverrides": {
-			// "reportImplicitStringConcatenation": "warning",
-			// "reportUnboundVariable": "warning",
-			// "reportDuplicateImport ": "warning",
-			// "reportUnusedClass": "information",
-			// "reportUnusedFunction": "information",
-			// "reportUnusedImport": "information",
-			// "reportUnusedVariable": "information",
+		"python": {
+			"analysis": {
+				// Offer auto-import completions.
+				"autoImportCompletions": true,
+				// Automatically add common search paths like 'src'?
+				"autoSearchPaths": true,
+				// Additional import search resolution paths
+				"extraPaths": [
```

Comment:
```json

		"python.analysis.extraPaths": [
			// For Sublime Text plugin development
			"$sublime_py_files_dir",
			"$packages",
		],
```

is removed here?

Reply:
yes LSP3 does not support them iirc.

Reply:
But this removal changes the current behavior of ST 4.